source/reference/operator/aggregation/planCacheStats.txt

```diff
@@ -175,24 +180,36 @@ including:
    * - ``timeOfCreation``
      - Time of creation for the entry.
 
-   * - ``creationExecStats``
+   * - :ref:`creationExecStats <plancachestats-creationExecStats>`
+
+     - .. _plancachestats-creationExecStats:
 
-     - An array of execution stats documents. The array contains a
+       An array of execution stats documents. The array contains a
        document for each candidate plan.
 
        For details on the execution stats, see
        :data:`explain.executionStats`.
 
-   * - ``candidatePlanScores``
+   * - :ref:`candidatePlanScores <plancachestats-candidatePlanScores>`
 
-     - An array of scores for the candidate plans listed in the
+     - .. _plancachestats-candidatePlanScores:
+
+       An array of scores for the candidate plans listed in the
        ``creationExecStats`` array.
 
    * - ``indexFilterSet``
 
      - A boolean that indicates whether the an :ref:`index filter
        <index-filters>` exists for the query shape.
 
+   * - ``estimatedSizeBytes``
+
+     - A number that describes the estimated size in bytes of a plan
+       cache entry.
+
+       .. versionadded:: 4.2.12 (and 4.0.23, and 3.6.23*)
```

Comment:
Looks like there might be a stray asterisk after 3.6.23?

Reply:
Oh there's actually a missing one... I'll fix it :)